_posts/2017-09-01-the-scourge-of-silos.markdown

```diff
@@ -0,0 +1,57 @@
+
+Wouldn't it be a better situation that the team responsible for the application actually create the infrastructure required as they are building it, using those techniques? The knowledge stays within that team, the architecture is emergent and the power to make positive change is in their own hands. 
+
+So why do organisations create a DevOps Team, which sits in its own bubble and has its own reporting structure, which essentially look after infrastructure, sometimes as code, in my experience? I think a lot of it comes down to organisational structure and command and control leadership styles. 
+
+![Command and Control Management Picture](/img/commdandcontrol.png)
+
+Command and control assumes a leader knows best and that they know where the organisation is going (goals, outcomes) and have a plan for how to get there (process). With a term as fuzzy as "DevOps", as illustrated by Josh's post, should we expect IT leaders in enterprise organisations, who have gone from Waterfall to Agile and now DevOps to be in touch enough to understand the nuance of it? Not in my experience. 
+
+So enter matrix management, which was designed to solve all these problems, by creating product centric delivery teams, but still retaining line management.
+
+![Matrix Management Picture](/img/matrix-man.png)
+
+The problem with matrix manangement is that fosters all sorts of problems. Political bullshit, being one. In this scenario, a team of individuals from different parts of the organisation are brought together to deliver something. In theory, this is great, but the problem is that line management for these indivduals sits off to one side. Any successes will be attributed to the members of that line manager and inversely any failures will be as a result of the other team members not aligning with that manager. The biggest problem, as a result of this, is the time to make decisions. It effectively widens the feedback loop (again) by involving interested third parties and will disenfranchise the team actually doing the work.
+
+The modern equivalent of matrix management in IT teams is the Spotify Model. Not that is actually a model. But expensive consultancies are paid loads to roll out a "model" which effectively propogates this shit. Guilds and tribes may have been formed with good intentions, but are just abused by the middle management that become the issue. 
```

Comment:
🤔

Reply:
Do you want to mention "there is no Spotify model" :)

Reply:
why chin scratch??